lib/pltr/v2/reducers/root.js

```diff
@@ -119,7 +130,8 @@ const root = (dataRepairers) => (state, action) => {
       )
       return mainReducer(state, newAction)
     }
-    case ATTACH_TAG_TO_CHARACTER: {
+    case ATTACH_TAG_TO_CHARACTER:
```

Comment:
@Quiescent im not sure why `ATTACH_TAG_TO_CHARACTER` and `DELETE_TAG` has the same reducer :sweat_smile:

Reply:
That's in the root reducer.  In that reducer, we add some extra detail to various actions before they reach specific reducers.
In the case of these new attributes, the downstream reducer needs to know whether we needed to make a new attribute, what the current book id is and sometimes what all the available attribute ids are.